quedex_api/user_stream.py

```diff
@@ -429,13 +418,21 @@ def send_batch(self):
     self._batch = None
     self._batch_mode = None
 
-  def time_triggered_batch(self, batch_id, execution_start_timestamp, execution_expiration_timestamp, order_commands):
+  def time_triggered_batch(self, timer_id, execution_start_timestamp, execution_expiration_timestamp, order_commands):
     """
-    Creates timer in exchange engine with a batch of order commands. These commands will be automatically
-    processed, between executionStartTimestamp and executionExpirationTimestamp.
-    Identifier of the created timer is the same as batch_id.
+    Sends a time triggered batch with the given list of order commands to the exchange.
+    When a time triggered batch is received by the exchange engine, a new timer is registered.
+    Based on the timer configuration, at some point in the future (between executionStartTimestamp and executionExpirationTimestamp),
+    all the carried order commands are processed, one by one, in the creation order. If executionStartTimestamp is in the past
+    and executionExpirationTimestamp is in the future, the batch will be processed immediately.
+
+    Timers are ordered by executionStartTimestamp and order of arrival. Timers with earlier executionStartTimestamp are processed first.
+    When there are more timers with the same executionStartTimestamp they are triggered by the order of arrival.
+
+    Note: timers are evaluated each time a new command is processed by the exchange engine. If between executionStartTimestamp
```

Comment:
As in java - unnecessary implementation detail. And what's important from user perspective - min. granularity of execution and why expiration is relevant.

Reply:
Removed. Added link to native API documentation.